examples/google-compute-engine/README.md

```diff
@@ -1,63 +1,85 @@
 # Example: Google Compute Engine
 
-Setting up a cluster in Google Compute Engine (GCE) might seem trivial at first, however
-if youtry to understand how it works exactly, if not that easy to read. This example is
-aimed to demonstrate how to deploy Kubernetes in GCE through a smallest number of steps.
+Although Google Compute Engine (GCE) may appear simple on the surface, configuring clusters is not a trivial task. With Kubernetes Anywhere, this learning curve is significantly shortened, enabling you to deploy clusters in GCE in just a few steps.
 
-Being part of Kubernetes Anywhere project, this example will utilise Wevae Net as management
-and apps network and all-Dockerized cluster components, thereby making it more portable
-and much easier to setup and operate.
+Kubernetes Anywhere makes use of the underlying technology in Weave Net and uses it as both a management and an apps network to deploy Dockerized cluster components. The result is a portable and simple way to configure and manage clusters onto any cloud provider.
```

Comment:
OK, we cannot just replace "deploy" with "bootstrap", this line needs a little bit of rewording...